easybuild/easyconfigs/i/iomkl/iomkl-2017.01-GCC-5.4.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2017.01'
+gcc_maj_min = '5.4'
+versionsuffix = '-GCC-%s' % gcc_maj_min
```

Comment:
@akesandgren we should consider dropping this `versionsuffix`...

I would keep it for exceptions, e.g. sticking to `GCC` `4.9.3` (which is quite old by now)